src/js/start.js

```diff
@@ -52,7 +52,7 @@ $(function() {
     $('[data-' + componentName + ']').politespace();
   });
 
-  $('[class^=usa-accordion]').each(function() {
+  $('*[class*=usa-accordion]').each(function() {
```

Comment:
Ah I remember now. 

I'm of the opinion that `$('.usa-accordion,.usa-accordion-bordered')` is a better solution. This is because with our CSS methodology, we could have classes like `.usa-accordion-child` that wouldn't represent the whole accordion, but an *element* of the accordion.

I don't see us creating that many different accordion types, and I think I'd be expected that if you did, you'd expect to have to add a selector for a new accordion type.